aten/src/ATen/core/dispatch/OperatorEntry.cpp

```diff
@@ -362,6 +362,19 @@ std::string OperatorEntry::listAllDispatchKeys() const {
   return str.str();
 }
 
+void OperatorEntry::reportSignatureError(std::string name) const {
```

Comment:
`std::string` by value? (eyebrow quirk)

Reply:
It's coming from a prvalue so there's no point in trying to pass that by reference -- https://www.internalfb.com/intern/diffusion/FBS/browsefile/master/fbcode/caffe2/aten/src/ATen/core/dispatch/CppSignature.h?commit=5d6752aa058ea0b4b2241ff90efc86d9d38c06d3&lines=28-30

See https://godbolt.org/z/6r6c38